MapboxNavigation/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -2,12 +2,19 @@
 
 @interface MGLMapView (MGLNavigationAdditions) <CLLocationManagerDelegate>
 
-// FIXME: This will be removed once https://github.com/mapbox/mapbox-gl-native/issues/6867 is implemented
-- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray<CLLocation *> *)locations;
+- (void)_setCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate
+                 edgePadding:(UIEdgeInsets)insets
+                   zoomLevel:(double)zoomLevel
+                   direction:(CLLocationDirection)direction
+                    duration:(NSTimeInterval)duration
+     animationTimingFunction:(nullable CAMediaTimingFunction *)function
+           completionHandler:(nullable void (^)(void))completion;
 
-// FIXME: This will be removed once https://github.com/mapbox/mapbox-navigation-ios/issues/352 is implemented.
-- (void)validateLocationServices;
+- (void)setCenterCoordinate:(CLLocationCoordinate2D)centerCoordinate
```

Comment:
Might be confusing this with another method, but don't we have this in native now?

Reply:
We added setCamera that takes edge padding. This was just during development. I think we can delete MGLMapView+MGLNavigationAdditions.h now.